File: get_feature_274.js
const Database = require('better-sqlite3');
const db = new Database('features.db', { readonly: true });

const feature = db.prepare('SELECT id, category, name, description, steps, passes, in_progress FROM features WHERE id = ?').get(274);

if (feature) {
    console.log('ID:', feature.id);
    console.log('Category:', feature.category);
    console.log('Name:', feature.name);
    console.log('Description:', feature.description);
    console.log('Steps:', feature.steps);
    console.log('Passes:', feature.passes);
    console.log('In Progress:', feature.in_progress);
} else {
    console.log('Feature #274 not found');
}

db.close();

File: testing-docs/get_feature_274.py
import sqlite3
import json

conn = sqlite3.connect('features.db')
cursor = conn.cursor()
cursor.execute('SELECT id, category, name, description, steps, passes, in_progress FROM features WHERE id = 274')
row = cursor.fetchone()

if row:
    print(f"ID: {row[0]}")
    print(f"Category: {row[1]}")
    print(f"Name: {row[2]}")
    print(f"Description: {row[3]}")
    print(f"Steps: {row[4]}")
    print(f"Passes: {row[5]}")
    print(f"In Progress: {row[6]}")
else:
    print("Feature #274 not found")

conn.close()
